homework/H2_Linear_Algebra.jl

```diff
@@ -729,6 +722,54 @@ md"How does a high/low bandwidth affect the prediction? How does regularization
 You can also implement [more kernel functions](https://en.wikipedia.org/wiki/Positive-definite_kernel#Examples_of_p.d._kernels) and pass them to the `kernel_ridge` keyword-argument `kernel` to see how the prediction changes.
 "
 
+# ╔═╡ 41a7bd3b-56f5-45e7-ac8d-313bb3a46181
+md"### Exercise 4 – Principal Component Analysis
+Note: This exercise does not provide automated feedback"
```

Comment:
Maybe we should explain why, e.g.:
"This exercise does not provide automated feedback to get you to practice programming on your own."

Reply:
Good call! That will be changed in the following commit!